nano/node/ipc/ipc_server.hpp

```diff
@@ -1,6 +1,5 @@
 #pragma once
 
-#include <nano/ipc_flatbuffers_lib/generated/flatbuffers/nanoapi_generated.h>
```

Comment:
10 lines down, this code can also be removed, which is likely why this include existed in the first place:
namespace flatbuffers
{
class Parser;
}